test/ampAphroditeInterfaceFactory_test.js

```diff
@@ -98,4 +100,44 @@ describe('ampAphroditeInterfaceFactory', () => {
       expect(aphroditeInterfaceResolveRTLSpy.callCount).to.equal(1);
     });
   });
+
+  describe('isAmp set to true', () => {
+    beforeEach(() => {
+      sinon.stub(isAmp, 'default').returns(true);
```

Comment:
Just calling out that this approach will break of babel ever complies with the spec here.

Reply:
Yeah, I know but otherwise this isn't super testable.